src/utils/config.ts

```diff
@@ -28,16 +32,27 @@ const connectors = connectorsForWallets(
   },
 );
 
+const injectedConnector = unstable_connector(injected);
+
+// RPC fallback order: Injected RPC > Alchemy RPC > Public RPC
+const transportsObj: Record<[wagmiChains.Chain, ...wagmiChains.Chain[]][number]['id'], Transport> = Object.fromEntries(
+  Object.entries(alchemyUrls).map(([chainId, url]) => [chainId, fallback([injectedConnector, http(url), http()])]),
+);
+
+// const isDevEnvironment = !!process && process.env.NODE_ENV === 'development';
+
+// const transports = isDevEnvironment ? { ...transportsObj, [localhost.id]: injectedConnector } : transportsObj;
+const transports = transportsObj;
+// export const availableChains = isDevEnvironment ? [localhost, ...supportedChains] : [...supportedChains];
+export const availableChains = supportedChains;
+
 export const config = createConfig({
-  chains: [localhost, sepolia],
+  chains: availableChains as [Chain, ...Chain[]],
```

Comment:
This name change seems unnecessary imo (same with `transports`)
```suggestion
export const config = createConfig({
  chains: supportedChains as [Chain, ...Chain[]],
```

Reply:
I only left testnet for now. I will try to implement localhost by the end of the challenge